runtime/onert/backend/cpu/ops/AddLayer.cc

```diff
@@ -84,12 +84,51 @@ void AddLayer::addQuant8()
   int32_t output_activation_min, output_activation_max;
   CalculateActivationRangeUint8(_activation, _output, &output_activation_min,
                                 &output_activation_max);
-  // nnfw::cker::BinaryArithmeticOpParam op_params;
-  // op_params.quantized_activation_max = output_activation_max;
-  // op_params.quantized_activation_min = output_activation_min;
+  nnfw::cker::BinaryArithmeticOpParam op_params;
+  op_params.type = nnfw::cker::BinaryArithmeticOpType::ADD;
+  op_params.quantized_activation_max = output_activation_max;
+  op_params.quantized_activation_min = output_activation_min;
+  // Parameters for scaled quantized computation
+  op_params.left_shift = 20;
+  // Zero-points of input and output tensors
+  op_params.input1_offset = _lhs->data_offset();
+  op_params.input2_offset = _rhs->data_offset();
+  op_params.output_offset = _rhs->data_offset();
```

Comment:
I think **output_offset** must be **_output->data_offset()** right ?

Reply:
yes, right :), thanks for spotting. Will address this.